frontend/device/pocketbook/device.lua

```diff
@@ -62,6 +62,10 @@ local PocketBook = Generic:extend{
     --- @fixme: Never actually set anywhere?
     is_using_raw_input = nil,
 
+    -- InkView may have started translating button codes based on rotation on newer devices...
+    -- That historically wasn't the case, hence this defaulting to false.
+    iv_translates_buttons = false,
```

Comment:
You could prefix these with `inkview_` instead of the less explicite `iv_` ?

Reply:
It also happens to be the standard prefix for iv functions, which is more of a reason *not* to use it here, ;).